install.sh

```diff
@@ -436,21 +457,28 @@ if [ $upgrade == true ]; then
                     LEFT JOIN information_schema.referential_constraints rc ON rcu.CONSTRAINT_NAME = rc.CONSTRAINT_NAME \
                     WHERE rcu.TABLE_SCHEMA = '$DB_NAME' AND rcu.CONSTRAINT_NAME LIKE 'iSkyLIMS_%' \
                     GROUP BY rcu.TABLE_SCHEMA, rcu.TABLE_NAME, rcu.CONSTRAINT_NAME, tc.CONSTRAINT_TYPE, rcu.REFERENCED_TABLE_SCHEMA, rcu.REFERENCED_TABLE_NAME, rc.DELETE_RULE;"
-            mysql -u $DB_USER -p$DB_PASS -h $DB_SERVER_IP -e "$query_rename_constraints" | xargs -I % echo "mysql -u$DB_USER -p$DB_PASS -D $DB_NAME -h $DB_SERVER_IP -e \"% \" " | bash
+            mysql -u $DB_USER -p'$DB_PASS' -h $DB_SERVER_IP -e "$query_rename_constraints" | xargs -I % echo "mysql -u$DB_USER -p$DB_PASS -D $DB_NAME -h $DB_SERVER_IP -e \"% \" " | bash
 
             echo "Done modifying database names and constraints..."        
-            # rm -rf ./*/migrations/__pycache__
-            # rm -rf ./*/migrations/__init__.py
-            
+
             # copy modified migration files
             echo "Copying custom migration files from conf."
-            ./manage.py makemigrations --empty -n migration_v2_3_1 core drylab wetlab django_utils
```

Comment:
esto pruébalo, estoy casi segura que es necesario para que la migration se cree en la base de datos y luego se haga: [https://stackoverflow.com/questions/50320989/how-would-you-create-a-manual-django-migration](https://stackoverflow.com/questions/50320989/how-would-you-create-a-manual-django-migration)

Reply:
--empty crea solo esto, que yo creo que no nos sirve para nada
![image](https://github.com/BU-ISCIII/iSkyLIMS/assets/24755054/ac0c84e0-c588-456b-981d-c1986829c945)